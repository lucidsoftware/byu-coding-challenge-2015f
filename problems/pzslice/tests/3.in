50 50
2 6
8 9
4 9
7 8
4 10
9 10
9 3
6 3
2 1
1 2
4 6
9 10
7 1
7 4
6 5
9 8
2 3
2 1
5 4
9 4
10 1
4 1
3 3
6 5
9 1
5 8
7 10
9 6
8 3
3 8
5 7
5 1
1 10
6 8
3 10
7 1
2 4
10 8
2 2
4 1
4 4
3 6
9 3
9 1
3 8
5 10
4 9
8 6
3 4
4 5