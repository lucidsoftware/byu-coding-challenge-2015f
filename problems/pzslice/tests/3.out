19.9999237061
